web/modules/custom/dpl_event/src/EventWrapper.php

```diff
@@ -86,6 +86,22 @@ private function getDate(string $value): \DateTimeInterface {
     return new DateTimeImmutable($event_date_values[$value], new \DateTimeZone('UTC'));
   }
 
+  /**
+   * Getting an events branches.
+   *
+   * @return array<NodeInterface>|null
+   *   The matching branches.
+   */
+  public function getBranches(): ?array {
```

Comment:
I've created a single method, that can be used both by Redia and the RestMapper.

Reply:
:+1: Excellent! I appreciate you updating it both places.